MekHQ/src/mekhq/campaign/finances/Finances.java

```diff
@@ -331,7 +331,7 @@ public void newDay(Campaign campaign) {
                             String.format(resourceMap.getString("Salaries.text"),
                                     payRollCost.toAmountAndSymbolString()));
 
-                    if (campaign.getCampaignOptions().trackTotalEarnings()) {
+                    if (campaign.getCampaignOptions().isTrackTotalEarnings()) {
```

Comment:
In terms of english language syntax, it's pretty awkward.

Reply:
It's a style thing... is means it's a boolean value, otherwise get is used.

Reply:
Sure, but when you're reading it in English, the sentence structure is pretty awkward.